cmd/login/cmd.go

```diff
@@ -121,6 +122,15 @@ func init() {
 		"Enables insecure communication with the server. This disables verification of TLS "+
 			"certificates and host names.",
 	)
+	flags.StringVar(
+		&args.rhRegion,
+		"rh-region",
+		"",
+		fmt.Sprintf(
```

Comment:
@tirthct You don't need the `fmt.Sprintf()` here as there is no formatting on the string.

Reply:
Ah, yes. Removed.